### DISCUSSION TITLE
[Question]: Is there Camera Loss event ?

### DISCUSSION THREAD

**Discussion by rdkartono** [blakeblackshear/frigate#11844] [Ask A Question] [answered]
### What is your question:

following my previous problem in [another post](https://github.com/blakeblackshear/frigate/discussions/11819), I want to know if Frigate + HA give camera loss event ? I think if there is camera loss event, it is better to solve my previous problem by this event, instead of running cron every 1 minutes and updating /etc/hosts file.

**Comment by hawkeye217** [comment, marked as answer]
For users running the Frigate HA integration before version 5.0, check the `camera_fps` sensor in HA for your camera. If it's `0`, your camera is offline.

For 5.0 and later, if your camera entity is `unavailable`, the camera is offline.
